cla-backend-go/v2/user-service/client.go

```diff
@@ -0,0 +1,61 @@
+package user_service
+
+import (
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/communitybridge/easycla/cla-backend-go/token"
+	"github.com/communitybridge/easycla/cla-backend-go/v2/user-service/client"
+	"github.com/communitybridge/easycla/cla-backend-go/v2/user-service/client/bulk"
+	"github.com/communitybridge/easycla/cla-backend-go/v2/user-service/models"
+	runtimeClient "github.com/go-openapi/runtime/client"
+	"github.com/go-openapi/strfmt"
+)
+
+// Client is client for user_service
+type Client struct {
+	cl        *client.UserService
+	dummyXACL string
+}
+
+var (
+	userServiceClient *Client
+)
+
+// InitClient initializes the user_service client
+func InitClient(APIGwURL string) {
+	APIGwURL = strings.ReplaceAll(APIGwURL, "https://", "")
+	userServiceClient = &Client{
+		cl: client.NewHTTPClientWithConfig(strfmt.Default, &client.TransportConfig{
+			Host:     APIGwURL,
+			BasePath: "user-service/v1",
+			Schemes:  []string{"https"},
+		}),
+		dummyXACL: "ewogICAgImFsbG93ZWQiOnRydWUsCiAgICAiaXNBZG1pbiI6IHRydWUsICAgIAogICAgInJlc291cmNlIjoidmlld19wcm9qZWN0IiwKICAgICJzY29wZXMiOlsgCiAgICBdIAp9",
```

Comment:
Offf, magic code alert.  I assume this is to set the ACL header?  If needed, let's externalize this as a SSM parameter or something - not hard-coded in the code.

Reply:
Removed it.